### ISSUE TITLE
Axels super fancy and unnecessary not using Thread.start() issue

### ISSUE THREAD

**Issue by Geosearchef** [FAForever/downlords-faf-client#831]
Sometimes this create an issue and branch stuff gets in the way of getting stuff done. 

@axel1200 here you go

**Comment by 1-alex98** [comment]
👍 i like

**Comment by micheljung** [comment]
So what do you propose instead?

**Comment by Geosearchef** [comment]
Is there no better way to do this? CompeltableFuture.runAsync? Do we have a service for thread pooling?

**Comment by 1-alex98** [comment]
Shell I do it in another PR  @micheljung ?

**Comment by micheljung** [comment]
To my understanding, a thread pool is meant to be used for short-lived tasks to be executed in parallel. In this case, however, we're spawning a replay server that 1. Needs its own thread and 2. Runs for a long time. It's like a sub-program. If we submit multiple such long-living tasks to the thread pool, the thread pool will eventually block if it runs out of workers.

This is the reason why I think it's better to create one explicit thread which will terminate when the replay server closes. I'm open to counter-arguments.

**Comment by Geosearchef** [comment]
Ok, good point
